fishjam/_ws_notifier.py

```diff
@@ -19,63 +32,64 @@
     ServerMessageSubscribeResponse,
 )
 
+ALLOWED_NOTIFICATION = (
```

Comment:
I think we should have a single place (module/ file) where the allowed notifications are stored - I don't see scenario where there are different ones for webhook and WsNotifier